model/release.go

```diff
@@ -64,15 +65,30 @@ func (tags Tags) MarshalJSON() ([]byte, error) {
 
 func (tags *Tags) Dedup() {
 	// Deduplicate tags:
-	set := make(map[Tag]bool)
-	result := []Tag{}
-	for _, item := range *tags {
-		if _, exists := set[item]; !exists {
-			set[item] = true
-			result = append(result, item)
+	s := *tags
```

Comment:
it's `s` for `someTags` right? `s` for `slice` would of course bind this to the implementation and not be very fitting if this was ever changed to a `map`